guides/integrations-api/general-concepts/README.md

```diff
@@ -20,5 +20,8 @@ And a common approach regarding the way that Shopware provides endpoint referenc
 
 {% page-ref page="generated-reference.md" %}
 
+## API Versioning
 
 
+
+{% page-ref page="api-versioning.md" %}
```

Comment:
Any idea what I can write here? 😄

Reply:
`Starting with Shopware version 6.4.0.0, we decided to change our API versioning strategy. 
The following article will cover what has been done and changed, how it used to be and how the version strategy looks like now.`


Copied it from the article itself and modified a bit.